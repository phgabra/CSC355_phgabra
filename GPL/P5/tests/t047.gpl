// make sure you are not using an int when you implement double division
double x = 3.63;
double y = 3.3;
double z = x / y;
