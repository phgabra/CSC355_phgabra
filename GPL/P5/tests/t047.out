gpl.cpp::main()
  input file(tests/t047.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double x = 3.63
double y = 3.3
double z = 1.1
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
